### ISSUE TITLE
`no-invalid-aria-attributes` fails when it is passed a valid if statement

### ISSUE THREAD

**Issue by gvdp** [ember-template-lint/ember-template-lint#2451]
We have a component in the code which has a different (translated) aria-label based on some check:

```
  <button
    aria-label={{if @isNew (t "actions.add") (t "actions.edit")}}
    ....
  />
  ```
  I think this should pass the linting check since there will always be a valid string assigned to the aria-label attribute?
  
  I tried this case in the existing unit test for this rule and the test indeed fails, I can try to come up with a proposal for a fix but I'm not familiar with this linting code base.

**Comment by bmish** [comment]
What violation message does it fail with?

**Comment by gvdp** [comment]
Thanks for the quick response!

"The value for aria-label must be a string." is the error message.

**Comment by bmish** [comment]
@judithhinlung what's your position on allowing:

1. Simple if statements like this
2. The common `t` translation helper in lieu of an actual string

**Comment by judithhinlung** [comment]
I agree that these are all valid inputs. I can update the rule so that it accepts   helpers declared in mustache statements.
